### PR TITLE
changes to search feature results

### PR DESCRIPTION
**Pull request by vibhiquartech**

### CODE REVIEW

File: django/api/admin.py [hunk 1]
```diff
@@ -35,6 +35,29 @@ def has_delete_permission(self, request, obj=None):
...
+        "middle_names",
+        "email",
+        "user_is_bcsc",
+        "address",
+        "city",
+        "postal_code",
+        "drivers_licence",
+        "date_of_birth",
+        "tax_year",
+        "doc1",
```
Comment (tim738745) [inline django/api/admin.py:52]: you can get rid of doc_1 and doc_2 since we're using the doc1_tag and doc2_tag

File: django/api/admin.py [hunk 2]
```diff
@@ -117,6 +140,7 @@ def message_user(
 
 @admin.register(GoElectricRebate)
 class GoElectricRebateAdmin(admin.ModelAdmin):
+    exclude = ("sin",)
```
Comment (tim738745) [inline django/api/admin.py:143]: I don't believe GoElectricRebate has a SIN field, so this can be deleted

**Review by tim738745 — CHANGES_REQUESTED** [review]

File: django/api/admin.py (hunk 1 above)
Comment (tim738745) [reply django/api/admin.py:52]: also, you probably want to include doc1 and doc2 in the excluded_fields tuple

Comment (vibhiquartech) [reply django/api/admin.py:52]: shouldn't the docs be visible in the application?

**Review by tim738745 — APPROVED** [review]

**Pull request merged**